pages/blog/google-season-of-docs-2022.md

```diff
@@ -0,0 +1,95 @@
+We will measure success in Docs project’s success partially by capturing specific feedback about the IA changes via our soon-to-come new [Docs Feedback card](https://github.com/asyncapi/website/issues/453). We need this specific and granular feedback to make sure we listen and make changes according to what the community requests from Docs. In previous AsyncAPI Docs Gist updates, we've mentioned that Design contributors were teaming with Docs on `/website`issue [#453](https://github.com/asyncapi/website/issues/453) for the ideation and development of our new **feedback card** that will be added at the bottom of each Docs page. What the community decided over the last 2 weeks was that the `Submit feedback` button in the card will publish the feedback anonymously via the AsyncAPI bot and create a new **GitHub Discussion** with said feedback:
+
+<img width="500" alt="Screen Shot 2022-03-10 at 8 47 22 PM" src="https://user-images.githubusercontent.com/19964402/158282499-a50737c0-a669-417d-99ae-a91ab21a0c07.png">
+<img width="500" alt="Screen Shot 2022-03-10 at 8 48 47 PM" src="https://user-images.githubusercontent.com/19964402/158282500-a29c953b-e333-455f-b5bb-c1d2491fef2a.png">
+
+The other way we would consider the project successful is the number of our contributors and Docs PRs increased by even 2-5 more community members. Currently, a majority of our OSS contributor community focuses only on contributing code, but we would like to instill a greater interest in contributing to documentation that provides value for everyone.
+
+
+### ⌛ Timeline
+The project itself will take approximately 3-6 months to complete, depending on how many technical writers (TW) get involved. If we can get even just 2-4 TWs, then we can work on both migrating the CLI and Tools documentation. But if we can only get 1-2 folks, it would be hard to do more than the migration of CLI or Tools docs. Ideally, the timeline could look like the following:
+
+- **May:** Orientation on how to contribute to AsyncAPI Inititiave, how Docs issues are organized, detail how we're migrating our CLI and Tools Docs, and assign good `first-time-tickets` to get each new TW contributor started. 
+- **June - August:**	Each TW goes through designated issues marked for both first time contributors and work set aside for `GSoD 2022`. Each TW starts creating documentation for their individual issues assigned/selected. 
+- **September - October:** We determine if we're going to be able to complete both CLI and Tools Docs, depending on how many TWs are in our group and how much they've been able to complete so far. We re-align priorities as needed and asses what is missing to reach our 2022 IA change goals for AsyncAPI Docs.
+- **November:**	Project completion and all contributors receive some swag! 
+
+
+### 💸 Project budget
+We have set aside 2 mentors for now, for our 1 project on improving our IA. For this we would like to request a US $5000 budget to Google, should we be selected. 
```

Comment:
we need at least 2 projects. Preparing Generator that is one of our main tools cannot be a part of this project as it is big enough to be an independent project under GSoD. Current Generator docs need to be restructured and updated + we need a set of tutorials that explain every single functionality of the Generator basing on this example -> https://github.com/asyncapi/template-for-generator-templates

Reply:
Got it, then I will make this project our 2nd one and I will also update our budget accordingly :)